File: Chavda_23B0368/Week_1/Missing_number.java
package Week_1;
import java.util.*;

class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        long total = (long) n * (n + 1) / 2;
        long sum = 0;
        
        for (int i = 0; i < n - 1; i++) {
            sum += sc.nextInt();
        }
        
        System.out.println(total - sum);
        sc.close();
    }
}

File: Karan_24b3003/Week 1/missing_number.cpp
#include <iostream>
using namespace std;
int main() {
    int n,x;
    cin >> n;
    long long total = 1LL * n * (n + 1) / 2;
    long long sum = 0;
    for (int i = 0; i < n - 1; ++i) {
        cin >> x;
        sum += x;
    }
    cout << (total - sum) << endl;
    return 0;
}
